plugins/openstack/pyparts/network.py

```diff
@@ -69,6 +73,17 @@ def get_config_network_info(self):
             config_info['nova'][key] = "{} ({})".format(port.addresses[0],
                                                         port.name)
 
+        for key, port in self.octavia_bind_interfaces.items():
+            if 'octavia' not in config_info:
+                config_info['octavia'] = {}
+
+            if len(port.addresses) > 0:
+                config_info['octavia'][key] = "{} ({})".format(
+                        port.addresses[0], port.name)
+            else:
+                msg = ("Octavia Health manager port has no IP address")
```

Comment:
Lets elaborate this a bit e.g.:

"No IP address found on Octavia healthmanager port ({}). Octavia will not be able to communicate with Amphora vms - please investigate.".format(port.name)

Reply:
o-hm0 is hardcoded in charm-octavia https://opendev.org/openstack/charm-octavia/src/branch/master/src/lib/charm/openstack/octavia.py#L39